tools/packaging/guest-image/build_se_image.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 IBM Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[ -n "${DEBUG:-}" ] && set -x
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly script_name="$(basename "${BASH_SOURCE[0]}")"
+readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly packaging_root_dir="$(cd "${script_dir}/../" && pwd)"
+readonly kata_root_dir="$(cd "${packaging_root_dir}/../../" && pwd)"
+
+source "$kata_root_dir/ci/lib.sh"
+source "${packaging_root_dir}/scripts/lib.sh"
+
+[ "$(uname -m)" = s390x ] || die "Building a Secure Execution image is currently only supported on s390x."
+
+usage() {
+	cat >&2 << EOF
+Usage:
+  ${script_name} [options]
+
+Options:
+  --builddir=${builddir}
+  --destdir=${destdir}
+
+Environment variables:
+  HKD_PATH (required): Secure Execution host key document, generally specific to your machine. See
+                  https://www.ibm.com/docs/en/linux-on-systems?topic=tasks-verify-host-key-document
+                  for information on how to retrieve and verify this document.
+  DEBUG         : If set, display debug information.
+EOF
+	exit "${1:-0}"
+}
+
+build_image() {
+	image_source_dir="${builddir}/secure-image"
+	mkdir -p "${image_source_dir}"
+	pushd "${tarball_dir}"
+	for tarball_id in kernel rootfs-initrd; do
+		tar xvf kata-static-${tarball_id}.tar.xz -C "${image_source_dir}"
+	done
+	popd
+
+	protimg_source_dir="${image_source_dir}${prefix}/share/kata-containers"
+	local kernel_params="agent.enable_signature_verification=false"
```

Comment:
I don't think this feature is in main yet.

Reply:
Then, we could start with an empty string. Thanks for the info.